projects/power_distribution/test/test_fan_ctrl.c

```diff
@@ -0,0 +1,219 @@
+#include <stdbool.h>
+#include "adc.h"
```

Comment:
Nitpick: there should be a blank line between the <stdbool.h> and adc.h includes.